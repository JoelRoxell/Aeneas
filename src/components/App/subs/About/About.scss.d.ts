// This file is generated automatically
export const about: string;
