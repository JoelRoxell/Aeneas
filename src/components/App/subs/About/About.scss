.about {
  flex: 1;
  border: rem(3) solid cyan;
}
