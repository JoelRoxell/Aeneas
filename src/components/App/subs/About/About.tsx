import * as React from 'react'
import * as style from './About.scss'

interface AboutProps {
  title: string
}

class About extends React.Component<AboutProps> {
  render() {
    return (
      <div className={style.about}>
        <p>About</p>
      </div>
    )
  }
}

export default About
