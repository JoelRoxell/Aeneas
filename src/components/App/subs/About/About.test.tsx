import * as React from 'react'
import { shallow } from 'enzyme'
import About from 'components/App/subs/About'

describe('About', () => {
  test('render', () => {
    const wrapper = shallow(<About />)

    expect(wrapper).toMatchSnapshot()
  })
})
