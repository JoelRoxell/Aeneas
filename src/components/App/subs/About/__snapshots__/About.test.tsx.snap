// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About render 1`] = `
<ForwardRef>
  <h2>
    About
  </h2>
  <ForwardRef>
    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </ForwardRef>
  <ForwardRef>
    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
  </ForwardRef>
  <ForwardRef>
    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
  </ForwardRef>
  <ForwardRef>
    It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </ForwardRef>
  <Button
    title="login"
    to="/login"
  />
</ForwardRef>
`;
